standards/licencing-software-or-code.md

```diff
@@ -0,0 +1,58 @@
+---
+category: Building software
+expires: 2018-01-31
+---
+# Licensing software or code
+
+Everything we produce or use, whether open or closed, should have an
+appropriate licence applied (as [indicated by the service
+manual][service manual licencing]). Without a licence, others (even
+within MOJ) have no way of knowing for sure if they can use it.
+
+At the MOJ, we use the [MIT License](https://opensource.org/licenses/MIT).
+
+[service manual licencing]: https://www.gov.uk/service-manual/technology/making-source-code-open-and-reusable#licensing-your-code
+
+## Applying the MIT License to our work
+
+### Each repository should include a licence file
+
+This should be called `LICENCE` or `LICENCE.md`. "License" is the U.S.
+English spelling.
+
+GitHub will still show licence details for the British English spelling.
+
+Make sure the licence content is included in full, including the title
+"The MIT License", so that readers are quickly able to see what licence
+is being used.
+
+### Use the correct copyright notice
+
+The Copyright is Crown Copyright; you can put "Ministry of Justice" in
```

Comment:
... Copyright; optionally you can ...

Reply:
I think that's implied by “can”?

Reply:
I'm being way too pedantic. Some purists say that 'can' refers to a technical ability to do something, not the permission to do so. As a stupid analogy, I 'can' put "Ministry of Sound" in the copyright notice, but it is neither correct nor optional to do so in this context.